index.bs

```diff
@@ -438,15 +438,15 @@ results and is easy to implement in hardware, this is a common solution.
 
 ### Complementary filter ### {#complementary-filters}
 
-A <dfn>complementary filter</dfn> can be thought of as a low and [=high-pass filter=] in one,
+A <dfn>complementary filter</dfn> can be thought of as a [=low-pass filter=] and [=high-pass filter=] in one,
 complementing the [=gyroscope=] values with the [=accelerometer=] values:
 
 θ<sub>n</sub> = α × (θ<sub>n-1</sub> + ω × ∂t) + (1.0 - α) × a
 
 With α being the weight constant, a the acceleration from [=accelerometer=], ω the angular
 velocity from [=gyroscope=] and ∂t being the time between measurements.
 
-A common value for α is 0.98, which means that 98% of the weight lays on the [=gyroscope=]
+A common value for &alpha; is 0.98, which means that 98% of the weight lays on the [=gyroscope=]
```

Comment:
@kenchris This is now an html entity. Can we do a global change to html entitities in the spec, or do I need to ditch TextMate :P